packages/fxa-auth-server/lib/routes/session.js

```diff
@@ -368,21 +370,19 @@ module.exports = function(
           throw error.invalidOrExpiredOtpCode();
         }
 
+        // If a valid code was sent, this verifies the session using the `email-2fa` method.
+        // The assurance level will be ["pwd", "email"] or level 1.
```

Comment:
For future readers, it may be worth adding a note here to say that we do this first before verifying the account, because we want it to hit the DB before we send any push notifications.